client/core/trade.go

```diff
@@ -2834,12 +2835,21 @@ func (c *Core) sendRedeemAsync(t *trackedTrade, match *matchTracker, coinID, sec
 		if match.Side == order.Maker && match.Status < order.MatchComplete {
 			// As maker, this is the end. However, this diverges from server,
 			// which still needs taker's redeem.
-			match.Status = order.MatchComplete
+			if conf := match.redemptionConfs; conf > 0 && conf >= match.redemptionConfsReq {
+				match.Status = order.MatchConfirmed // redeem tx already confirmed before redeem request accepted by server
```

Comment:
This condition isn't updated with a `!isConfirmer || ` so there's a delay until the next tick gets to `confirmRedemption` with the newy set `RedeemSig`.  Should we add the confirmer condition here too or is the delay fine?

Reply:
Made it a fast path in https://github.com/decred/dcrdex/pull/2432/commits/f35a0c77866849bdbcf065da22a949e876e55f38.
Still tests fine.